content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -0,0 +1,656 @@
+---
+id: get-started-with-the-rules-engine
+themes: boost-your-productivity, catalog-settings
+title: Get started with the **rules engine**
+popular: false
+ee-only: true
+related: what-is-a-rule
+---
+
+The rules engine can radically boost your productivity in the PIM by automating actions. The rules are defined in a YML file that you can import in the PIM, using a dedicated job profile.   
+
+This feature is very, like, really powerful, but it can also be a bit complex to apprehend. Reading this article, you will find out what you can use it for and how it works.
+
+
+# Available actions
+
+As of today, 5 actions are available in the rules engine:
+- copy
+- add
+- set
+- remove
+- concatenate
+
+What are these actions for? Let's discover it :wink:
+
+## Copy
+
+The `Copy` action enables you to copy an attribute value into another one.
+
+The expected values are:
+ - `from_field`: the code of the attribute to be copied
+ - `to_field`: the attribute code the value will be copied into
+ - `from_locale`: the locale code of the value to be copied *(optional)*
+ - `from_scope`: the channel code of the value to be copied *(optional)*
+ - `to_locale`: the locale code the value will be copied into *(optional)*
+ - `to_scope`: the channel code the value will be copied into *(optional)*
+
+
+### Example
+
+You have a *scopable* and *localizable* attribute called `description`. You can copy its content from the `en_US` locale and the `print` channel to the `en_US` locale and the `e-commerce` channel. The action will be defined as follows:
+
+```YML
+  actions:
+      type: copy
+        from_field: description
+        from_locale: en_US
+        from_scope: print
+        to_field: description
+        to_locale: en_US
+        to_scope: ecommerce
+```
```

Comment:
Actions represent a list of actions. So this is an array (represents by a `-` in YML).  
And becareful, the line `type: copy` must be at same level as the from_field, etc... 

```suggestion
```YML
  actions:
      -
        type: copy
        from_field: description
        from_locale: en_US
        from_scope: print
        to_field: description
        to_locale: en_US
        to_scope: ecommerce
```

```